src/Goteo/Console/EventListener/ConsoleWatcherListener.php

```diff
@@ -316,59 +308,25 @@ function ($a, $b) {
 
             // Extra
             case 26: // Send information about contract in order to prepare documentacion
```

Comment:
This case 26 will never be reached, since there's already one at line 232

Reply:
True. I think that the day 27 is free, do you think we can move it one day? @javicarrillo